File: 文档/实体类/Public/GroupMember.cs
using System;

namespace InstantMessaging.Core.Models.Public
{
    public class GroupMember
    {
        /// <summary>
        /// yonghuid
        /// </summary>
        public string UserId { get; set; }
        /// <summary>
        /// 群组角色
        /// </summary>
        public GroupMemberRole Role { get; set; }
        /// <summary>
        /// 最后一次活跃
        /// </summary>
        public DateTime LastActive { get; set; }
        /// <summary>
        /// 是否被禁言
        /// </summary>
        public bool IsMuted { get; set; }

    }
}

File: app/src/main/java/usung/com/mqttclient/bean/db/GroupMember.java
package usung.com.mqttclient.bean.db;

import java.io.Serializable;

/**
 * 群成员
 * @author herui
 * @date 2018/12/21
 */

public class GroupMember implements Serializable {
    /// <summary>
    /// yonghuid
    /// </summary>
    public long UserId ;
    /// <summary>
    /// 群组角色
    /// </summary>
    public int Role ;
    /// <summary>
    /// 最后一次活跃
    /// </summary>
    public String LastActive ;
    /// <summary>
    /// 是否被禁言
    /// </summary>
    public Boolean IsMuted ;

    public String JoinTime ;

    public long getUserId() {
        return UserId;
    }

    public void setUserId(long userId) {
        UserId = userId;
    }

    public int getRole() {
        return Role;
    }

    public void setRole(int role) {
        Role = role;
    }

    public String getLastActive() {
        return LastActive;
    }

    public void setLastActive(String lastActive) {
        LastActive = lastActive;
    }

    public Boolean getMuted() {
        return IsMuted;
    }

    public void setMuted(Boolean muted) {
        IsMuted = muted;
    }

    public String getJoinTime() {
        return JoinTime;
    }

    public void setJoinTime(String joinTime) {
        JoinTime = joinTime;
    }
}
